lib/rubocop/cop/layout/empty_lines_around_attribute_accessor.rb

```diff
@@ -4,6 +4,8 @@ module RuboCop
   module Cop
     module Layout
       # Checks for a newline after attribute accessor.
+      # `alias` syntax and `alias_method`, `public`, `protected`, and `private` methods are ignored by default.
```

Comment:
We should also be more specific in the initial sentence - it's an attribute accessor or a group of them.